<?php

namespace Container2YcVERS;

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

/**
 * @internal This class has been auto-generated by the Symfony Dependency Injection Component.
 */
class getEmailInvitationValidatorService extends App_KernelDevDebugContainer
{
    /**
     * Gets the private 'Ibexa\User\Validator\Constraints\EmailInvitationValidator' shared autowired service.
     *
     * @return \Ibexa\User\Validator\Constraints\EmailInvitationValidator
     */
    public static function do($container, $lazyLoad = true)
    {
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidatorInterface.php';
        include_once \dirname(__DIR__, 4).'/vendor/symfony/validator/ConstraintValidator.php';
        include_once \dirname(__DIR__, 4).'/vendor/ibexa/user/src/lib/Validator/Constraints/EmailInvitationValidator.php';

        $a = ($container->services['ibexa.api.service.user'] ?? $container->getIbexa_Api_Service_UserService());

        if (isset($container->privates['Ibexa\\User\\Validator\\Constraints\\EmailInvitationValidator'])) {
            return $container->privates['Ibexa\\User\\Validator\\Constraints\\EmailInvitationValidator'];
        }

        return $container->privates['Ibexa\\User\\Validator\\Constraints\\EmailInvitationValidator'] = new \Ibexa\User\Validator\Constraints\EmailInvitationValidator($a);
    }
}
